Documentation/Blazorise.Docs/Pages/News/2024-10-15-release-notes-170.razor

```diff
@@ -98,6 +98,10 @@
     Optimizations
 </Heading>
 
+<Paragraph>
+    We added options for Virtualized rendering to the TreeView component. This should help with large TreeViews.
```

Comment:
Very generic. What does Virtualize actually do? Maybe just extract a brief paragraph out of Microsoft docs.